x-pack/plugins/security_solution/public/resolver/data_access_layer/mocks/no_ancestors_two_children_with_related_events_on_origin.ts

```diff
@@ -78,8 +78,7 @@ export function noAncestorsTwoChildrenWithRelatedEventsOnOrigin(): {
        */
       async eventsWithEntityIDAndCategory(
         entityID: string,
-        category: string,
-        after?: string
```

Comment:
Removed unused param

Reply:
https://github.com/elastic/kibana/blob/0f70bdd41f610a40eb8f0b53d5e30cb8283f3bc1/x-pack/plugins/security_solution/public/resolver/data_access_layer/mocks/no_ancestors_two_children_with_related_events_and_cursor_on_origin.ts I think that was this guy